Update Debian packaging from Qt4/Harmattan to Qt5/Ubuntu

vranki opened the pull request:
This change allows building the Debian package on Ubuntu instead of obsolete Nokia Harmattan platform. 

It's not thoroughly tested, but "works for me". As a bonus there's a fix for missing unistd.h header which prevents compilation in some situation.

Commit:
Updated Debian packaging for Qt5 / Ubuntu.

Commit:
Merge branch 'master' of https://github.com/nemomobile/libqtsparql

thp commented:
LGTM.

vranki commented:
What is LGTM?

SfietKonstantin commented:
@vranki "Lookgs good to me", meaning that you can merge it.

vranki commented:
Great :)

amtep commented:
All this time I thought it was "looks good to merge"

vranki commented:
Btw, ubuntu has upgraded to tracker-sparql-1.0 and current master doesn't build with it. Are you planning to upgrade libqtsparql to support it some time? These dependencies need to be upgraded as well.

thp commented:
Our tracker in Nemo Mobile was updated to 1.1 here: https://github.com/nemomobile-packages/tracker/commit/8897d698cf1050646089e09aa389fa84a4831c8b

From http://packages.ubuntu.com/search?keywords=tracker it looks like Ubuntu has Tracker 1.0 in Utopic and Tracker 1.2 in Vivid.

I wonder of we could make it so that it works with both 1.1 and 1.2 or ideally with 1.0, 1.1 and 1.2 (as long as they are API-compatible) -- any suggestions?

Commit:
updated packaging

vranki committed:
Merge pull request #1 from jppiiroinen/master

updated packaging for cowbuilder-dist

Commit:
removed dependency to libqt4-dev

vranki committed:
Merge pull request #2 from jppiiroinen/master

removed dependency to libqt4-dev

Commit:
Support for 16.04 LTS.
Changed dependency from libtracker-sparql-0.16-dev to libtracker-sparql-1.0-dev.
Disabled doc package and compiling.
Added 'package' helper script for creating the deb package.

vranki committed:
Merge pull request #3 from jppiiroinen/master

Support for Ubuntu 16.04 LTS.
